packages/nodes-base/nodes/Salesforce/Salesforce.node.ts

```diff
@@ -243,6 +255,13 @@ export class Salesforce implements INodeType {
 				const returnData: INodePropertyOptions[] = [];
 				// TODO: find a way to filter this object to get just the lead sources instead of the whole object
 				const { fields } = await salesforceApiRequest.call(this, 'GET', '/sobjects/lead/describe');
+
+				for (const aja of fields as IDataObject[]) {
+					if (aja.custom === true) {
+						console.log(aja);
+					}
+				}
+
```

Comment:
@RicardoE105 I deleted this part. Does not make a difference as it just console.logs data. If the custom-check is needed in the function underneath, please create a PR. Thanks!